modules/tensor_mechanics/test/tests/global_strain/tests

```diff
@@ -1,44 +1,70 @@
 [Tests]
+  design = 'GlobalStrain/index.md'
+  issues = '#11314'
   [./test]
     type = 'Exodiff'
     input = 'global_strain.i'
     exodiff = 'global_strain_out.e'
+    requirement = "The globalstrain system shall check the volume change due to
+                   applied stress while still maintaining periodicity in 2D."
```

Comment:
Thanks for adding this stuff!  Just a minor thing:  The requirements should refer to what the code should be able to do and how it should behave, and the test just confirms that it behaves that way.

I would reword this as something like:

The globalstrain system shall correctly compute the volume change due to applied stress while still maintaining periodicity in 2D.

The same principle applies to the other tests that you've written requirements for.